lms/product/canvas/_plugin/course_copy.py

```diff
@@ -32,8 +33,21 @@ def find_matching_file_in_course(self, current_course_id, file_ids) -> str | Non
 
         Return None if no matching file is found.
         """
-        # Find the files in the current course. We call this for the side effect of storing the files in the DB
-        _ = self._api.list_files(current_course_id)
+        try:
+            # Find the files in the current course. We call this for the side effect of storing the files in the DB
+            _ = self._api.list_files(current_course_id)
+
+        except OAuth2TokenError:
```

Comment:
This is a verbatim copy of the "default" solution for this here:

https://github.com/hypothesis/lms/blob/main/lms/product/plugin/course_copy.py#L33


This PR applies the same solution in Canvas but doesn't address code duplication as there still some things that are Canvas-only here.

I'll leave that for a follow up refactor PR.